packages/mds-web-sockets/tests/test.ts

```diff
@@ -0,0 +1,30 @@
+import WebSocket from 'ws'
+import { MOCHA_PROVIDER_ID } from '@mds-core/mds-providers'
+import { PROVIDER_SCOPES } from '@mds-core/mds-test-data'
+import { WebSocketServer } from '../server'
+
+const ADMIN_AUTH = `basic ${Buffer.from(`${MOCHA_PROVIDER_ID}|${PROVIDER_SCOPES}`).toString('base64')}`
```

Comment:
I seem to recall a `SCOPED_AUTH` function that can be used to build authorization headers.